gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/reaper/InitializeReaper.kt

```diff
@@ -12,9 +12,11 @@ import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
 import org.gradle.api.tasks.StopExecutionException
 import org.gradle.api.tasks.TaskAction
+import org.gradle.work.DisableCachingByDefault
 import java.util.zip.ZipEntry
 import java.util.zip.ZipOutputStream
 
+@DisableCachingByDefault(because = "Reaper initialization should not be cached.")
```

Comment:
I think this also could be cacheable, assuming the artifact has had the transform run before and is 1:1 with a pre-ran task